src/init/update-artifacts/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.6'
 services:
 
   aeproject_node:
-    image: aeternity/aeternity:${NODE_TAG:-latest}-bundle
+    image: aeternity/aeternity:${NODE_TAG:-v6.5.2}-bundle
```

Comment:
https://github.com/aeternity/aeproject/pull/432#discussion_r953881755 ☝️

Reply:
thank you @davidyuk !